image-widget.php

```diff
@@ -573,8 +573,8 @@ private function upgrade_nag_plugins_admin_msg() {
 
 		$msg = sprintf(
 			__( '<p class="dashicons-before dashicons-format-gallery"><strong>Image Widget Plus</strong> - Add lightbox, slideshow, and random image widgets. <strong><a href="%s" target="_blank">Find out how!</a></strong></p>', 'image-widget' ),
-			'http://m.tri.be/19my',
-			'http://m.tri.be/19my'
+			'https://evnt.is/19my',
+			'https://evnt.is/19my'
```

Comment:
redundant second url?

Reply:
Indeed, there's only one placeholder.

```suggestion
			'https://evnt.is/19my'
```